osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -127,6 +124,15 @@ private void readBracket()
 
             if (addedInfo)
                 SaveChanges();
+
+            ladder.CurrentMatch.Value = ladder.Matches.FirstOrDefault(p => p.Current.Value);
+
+            Schedule(() =>
+            {
+                dependencies.Cache(ladder);
```

Comment:
Are you sure about this? `TournamentGame` derives from this class and adds `TournamentSceneManager` in its BDL which then adds all the scenes that resolve `LadderInfo` in its BDL. It seems like this is going to occur too late?

Reply:
In-fact it seems `TournamentSceneManager` itself has a dependency. I can't replicate every time but:

`osu.Framework.Allocation.DependencyNotRegisteredException: The type TournamentSceneManager has a dependency on LadderInfo, but the dependency is not registered.`

Reply:
Hmm interesting. Will need to move that inside the continuation callback, good catch.